packages/underscore-redirects/src/astro.ts

```diff
@@ -94,15 +100,15 @@ export function createRedirectsFromAstroRoutes({
 				}
 				_redirects.add({
 					dynamic: true,
-					input: pattern,
+					input: `${base}${route.pattern}`,
```

Comment:
what's the difference with `pattern` on line 89?

Reply:
`pattern` is for dynamic routes